build.gradle

```diff
@@ -93,3 +93,19 @@ check.dependsOn jacocoTestCoverageVerification
 springBoot {
 	buildInfo()
 }
+
+task startDb(type: Exec) {
+	commandLine "docker-compose", "up", "-d", "db"
+}
+
+task stopDb(type: Exec) {
+	commandLine "docker-compose", "down"
+}
+
+task startTestDb(type: Exec) {
```

Comment:
Not 100% sold on this naming pattern. Should it be testDbStart (and so on) instead?